python/hail/plot/plots.py

```diff
@@ -133,3 +211,56 @@ def qq(pvals):
     bound = max(max(exp), max(obs)) * 1.1
     p.line([0, bound], [0, bound], color='red')
     return p
+
+
+@typecheck(pvals=expr_float64, locus=nullable(expr_locus()), title=nullable(str), size=int)
+def manhattan(pvals, locus=None, title=None, size=4):
+    """Create a Manhattan plot. (https://en.wikipedia.org/wiki/Manhattan_plot)
+
+    Parameters
+    ----------
+    pvals : :class:`.Float64Expression`
+        P-values to be plotted.
+    locus : :class:`.LocusExpression`
+        Locus values to be plotted.
+    title : str
+        Title of the plot.
+    size : int
+        Size of markers in screen space units.
+
+    Returns
+    -------
+    :class:`bokeh.plotting.figure.Figure`
+    """
+    pvals = -hail.log10(pvals)
+
+    if locus is None:
+        locus = pvals._indices.source.locus
+    x = locus.global_position()
+    label = locus.contig
+
+    p = scatter(x, pvals, label=label, title=title, xlabel='Chromosome', ylabel='P-value (-log10 scale)',
+                size=size, legend=False)
+
+    ref = locus.dtype.reference_genome
+
+    total_pos = 0
+    mid_points = []
+    for i in range(0, 22):
```

Comment:
I don't think we can hardcode this as such. what happens if we just write it totally generically using `range(0, len(ref.lengths))`? All the grch38 contigs pile up on each other?